src/uct/ib/base/ib_log.c

```diff
@@ -64,7 +64,7 @@ void uct_ib_log_dump_sg_list(uct_ib_iface_t *iface, uct_am_trace_type_t type,
 
         s               += strlen(s);
 
-        if (data_dump) {
+        if ((i < data_dump_sge) && data_dump) {
```

Comment:
can we adjust `num_sge` in the callers instead of adding new param?

Reply:
need to distinguish between num sge to print va/len for, and num sge to print data for

Reply:
ok